_includes/repository.html

```diff
@@ -1,9 +1,17 @@
 <div id="repository">
-    <a class="button" href="https://github.com/cylc/cylc/releases/latest">
-        Download <span id="latest-release-name"></span><br />
-        <small>Released <time id="latest-release-pubdate-time" datetime=""><span id="latest-release-pubdate-text"></span></time></small>
+   <p>
+    <a class="button" id="cylc7-button" href="https://github.com/cylc/cylc-flow/releases">
+        <span id="cylc7-latest-release-name">(retrieving data...)</span>
+        <small>PRODUCTION<br /><time id="cylc7-latest-release-pubdate-time" datetime=""><span id="cylc7-latest-release-pubdate-text"></span></time></small>
     </a>
-    <p><a class="button" href="https://github.com/cylc/cylc">Clone cylc on GitHub</a></p>
+		</p>
+		<p>
+    <a class="button" id="cylc8-button" href="https://github.com/cylc/cylc-flow/releases">
+        <span id="cylc8-latest-release-name">(retrieving data...)</span>
+        <small>PREVIEW<br /><time id="cylc8-latest-release-pubdate-time" datetime="">
+								<span id="cylc8-latest-release-pubdate-text"></span></time></small>
```

Comment:
tabs/spaces? :grimacing:

Reply:
God damn it.  I need to fix my editor defaults on the work desktop machine. :grimacing:

Reply:
Done.